### ISSUE TITLE
python_installation object has no path() method

### ISSUE THREAD

**Issue by prlw1** [mesonbuild/meson#5034]
According to the [python_installation](http://mesonbuild.com/Python-module.html#python_installation-object) documentation, an [external program](http://mesonbuild.com/Reference-manual.html#external-program-object) is returned. External program apparently has a path() method. I tried this out on meson 0.49.2 with the following meson.build file:
```
project('mesonbug1', 'cpp')
pymod = import('python').find_installation()
python_name = pymod.path()
message('python path= ' + python_name)
```
but got the error
```
meson.build:3:0: ERROR:  Python object does not have method path.
```
Documentation error? How can I find the path to the python executable? (which obviously is found, as meson is running...)

**Comment by dcbaker** [comment]
Oops, I didn't update the docs when I fixed this. This is a known issue and is fixed in the upcoming 0.50 release. I'll send an update for the documentation too. See #4616 for the fix.

**Comment by dcbaker** [comment]
Here's the documentation fix: #5035

**Comment by prlw1** [comment]
Thanks!
